website/hw/hw1.rst

```diff
@@ -851,8 +850,11 @@ Now, ``git status`` will show the following:
         modified:   test.txt
 
 
-Go ahead and run ``git commit``. The commit will now include only
-``README.md``.
+Go ahead and commit this change::
+
+   git commit -m"Our last change?"
+
+The commit will now include only ``README.md``.
 
```

Comment:
Would we want to add something to the end of this section instructing students to delete the temp directory? I'm not sure how the tmp folder works/if this is even necessary.

Reply:
I may be mistaken, but I believe all /tmp directories will be cleaned upon reboot?

Reply:
Correct: the /tmp directories are clearead upon reboot (and, on the Linux servers, I believe they're also just periodically cleared because those servers tend to run for a long time without being rebooted)